docs/DeveloperGuide.md

```diff
@@ -641,14 +590,14 @@ _Customization of Contacts_
 
 <br/>
 
-**Use Case 13: View contact card**
+**Use Case 13: Pin contact card on the top of person list**
 
 *MSS*
 
-1.  User requests to **list contacts (UC02)** or **find contact (UC04)**
+1.  User requests to **list contacts (UC02)** or **find contact (UC03)**
 2.  SociaLite returns a list of contacts according to the UC called
-3.  User specifies the index of his desired contact
-4.  SociaLite opens a contact card which presents all previously stored details associated with the contact
+3.  User specifies contact to be pinned at the top of the list
+4.  SociaLite pins the contact at the top of the list
```

Comment:
Small nit here: maybe we should use "pin to the top..." to standardise with the use case title!

Reply:
Actually let me do it myself in a separate PR! Since this section should fall under me haha

Reply:
Roger boss!